src/util/Vocabularies.ts

```diff
@@ -148,6 +152,9 @@ export const SOLID_META = createUriAndTermNamespace('urn:npm:solid:community-ser
   'ResponseMetadata',
   // This is used to identify templates that can be used for the representation of a resource
   'template',
+  // This is used to store Content-Type Parameters
+  'ContentTypeParameter',
```

Comment:
```suggestion
  'contentTypeParameter',
```
In general, capitalized terms are only used for type values.